cairo/tests/ef_tests/cancun/test_state_transition.py

```diff
@@ -13,7 +13,10 @@
     run_blockchain_st_test,
 )
 
-pytestmark = pytest.mark.cairo_file(f"{Path().cwd()}/cairo/ethereum/cancun/fork.cairo")
+pytestmark = [
+    pytest.mark.cairo_file(f"{Path().cwd()}/cairo/ethereum/cancun/fork.cairo"),
+    pytest.mark.max_steps(100_000_000),
```

Comment:
max 100M steps per ef-test run: anything above is extremely likely to be >1B anyway, crashing the runner